app/views/admin/profiles/admin_profiles/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render TabsComponent.new do |c| %>
+  <%= c.tab(path: admin_admin_profiles_path) do %>
+    Admin Users
+  <% end %>
+  <%= c.tab(path: admin_induction_coordinator_profiles_path) do %>
+    Induction Coordinators Users
+  <% end %>
+  <%= c.tab(path: admin_lead_provider_profiles_path) do %>
+    Lead Providers Users
+  <% end %>
+<% end %>
+
+<table class="govuk-table">
+
+  <thead>
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Full Name</th>
+      <th scope="col" class="govuk-table__header">Email</th>
+      <th scope="col" class="govuk-table__header">Delete</th>
+    </tr>
+  </thead>
+
+  <tbody class="govuk-table__body">
+    <% @admin_profiles.each do |admin_profile| %>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell"><%= admin_profile.user.full_name %></td>
+        <td class="govuk-table__cell"><%= admin_profile.user.email %></td>
+        <td class="govuk-table__cell">
+          <% unless @current_user.admin_profile.id == admin_profile.id %>
+            <%= govuk_link_to "Delete User", admin_admin_profile_path(admin_profile.id), button: true, class: 'govuk-button--warning' %>
+          <% end %>
+        </td>
+      </tr>
+    <% end %>
+  </tbody>
+
+</table>
+<%= govuk_paginate @admin_profiles %>
+
```

Comment:
Do we have our own pagination components? I might have lost track of that.

Reply:
Yes, that's what it's called